commands.json

```
@@ -178,13 +178,26 @@
         "type": "key"
       },
       {
-        "name": [
-          "start",
-          "end"
-        ],
-        "type": [
-          "integer",
-          "integer"
+        "name": "index",
+        "type": "block",
+        "block": [
+          {
+            "name": "start",
+            "type": "integer"
+          },
+          {
+            "name": "end",
+            "type": "integer"
+          },
+          {
+            "name": "index_type",
```

```suggestion
            "name": "index_unit",
```

Bit/byte is the unit, isn't it? (Not important because it's not visible.)